src/compression/gzip/gzip-dowload.md

```diff
@@ -0,0 +1,35 @@
+## Decompress gzip response for a http request
+
+[![flate2-badge]][flate2] [![reqwest-badge]][reqwest] [![cat-decompression-badge]][cat-decompression]
+
+Donwload a content with [`reqwest`]. </br>
+[`GzEncoder`] is responsible for decompress the response.
+
+```rust,no_run
+# #[macro_use]
+# extern crate error_chain;
+extern crate reqwest;
+extern crate flate2;
+#
+# error_chain! {
+#     foreign_links {
+#         ReqError(reqwest::Error);
+#         Io(std::io::Error);
+#     }
+# }
+use std::io::Read;
+use flate2::read::GzDecoder;
+
+fn main() -> Result<()> {
+    let response = reqwest::get("https://httpbin.org/gzip")?;
+    let mut d = GzDecoder::new(response);
```

Comment:
please note that reqwest already has [gzip related capabilities](https://docs.rs/reqwest/0.8.6/reqwest/struct.ClientBuilder.html#method.gzip) if anything, we should show them as more idiomatic example.

Reply:
> 
> 
> please note that reqwest already has [gzip related capabilities](https://docs.rs/reqwest/0.8.6/reqwest/struct.ClientBuilder.html#method.gzip) if anything, we should show them as more idiomatic example.

Closing this PR and issue, there's nothing to show.